R/create_episode_file.R

```diff
@@ -138,13 +138,13 @@ create_episode_file <- function(
     episode_file <- episode_file %>%
       dplyr::mutate(
         sc_send_lca = NA,
```

Comment:
Is this variable is equivlent to sending_location? which variable should we use?

Reply:
This is a social care variable in the episode file. There is a function called `convert_sending_location_to_lca` which takes `sending_location` as a parameter and applies the local council authority codes to this. In the episode file we have `LCA` and `sc_send_lca` variables which is the social care equivalent. It aims to make the variables consistent by having a social care variable but basically uses sending location to convert into the LCA codes. Hope this helps

Reply:
Thanks for your reply! I add the variable `sc_send_lca` and `remove sending_location` in `join_sc_client_ep` function.